From 12a3818fda24892d40f4838e84b06fa36f48fbbf Mon Sep 17 00:00:00 2001
From: George Hopkins <george-hopkins@null.net>
Date: Thu, 28 Jun 2018 08:57:08 +0200
Subject: [PATCH] Relax test assertions

On some hosts (notably Travis), connecting to a unbound port returns
WebSocket::ErrorGeneric because the underlying socket returns
QAbstractSocket::UnknownSocketError.
---
 tests/websockettest/websockettest.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/websockettest/websockettest.cpp b/tests/websockettest/websockettest.cpp
index fdd40cd..ea60228 100644
--- a/tests/websockettest/websockettest.cpp
+++ b/tests/websockettest/websockettest.cpp
@@ -197,7 +197,7 @@ private slots:
 		sock.start(QString("http://localhost:1/"));
 		waitForSignal(&spy);
 
-		QCOMPARE(sock.errorCondition(), WebSocket::ErrorConnect);
+		QVERIFY(sock.errorCondition() != WebSocket::ErrorNone);
 	}
 
 	void handshakeSuccess()
-- 
2.7.4

